cmd/skaffold/app/cmd/build.go

```diff
@@ -82,6 +82,7 @@ func runBuild(out io.Writer) error {
 	if err != nil {
 		return err
 	}
+	runner.RPCServerShutdown()
```

Comment:
this should be `defer`red? and should probably return an error and log it if there is an issue?

Reply:
done